source/content.ts

```diff
@@ -135,6 +135,7 @@ import './features/pr-branch-auto-delete';
 import './features/show-whitespace';
 import './features/linkify-symbolic-links';
 import './features/hide-zero-packages';
+import './features/merge-status';
```

Comment:
The feature should probably be named `conflict-marker` or something along those lines

Reply:
Agreed. First I was thinking about displaying `merge status` as the icon and forgot to change the name.